doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -212,6 +212,9 @@ nRF Desktop
     The PDK can act as a sample mouse or keyboard.
     It supports the Bluetooth LE HID data transport and uses SoftDevice Link Layer with Low Latency Packet Mode (LLPM) enabled.
     The PDK uses MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) and supports firmware updates using the :ref:`nrf_desktop_dfu`.
+  * Support for the nRF54H20 DK with the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
+    The PDK can act as a sample mouse.
+    It supports the Bluetooth LE HID data transport and uses SoftDevice Link Layer with Low Latency Packet Mode (LLPM) enabled.
```

Comment:
```suggestion
  * Support for the nRF54H20 DK with the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
    The DK can act as a sample mouse.
    It supports the Bluetooth LE HID data transport and uses SoftDevice Link Layer with Low Latency Packet Mode (LLPM) enabled.
```
overalll lgtm, but please add it to the cs2 release notes: https://github.com/nrfconnect/sdk-nrf/pull/15273

Reply:
How should I add it to file which is not merged yet?

Reply:
Please add the entries applicable for the 2.6.99-cs2 release via a comment in Francesco's PR. This is a general remark. 
Release notes are always the last ones to get merged, so naturally the content needs to be updated prior to that.
Due to the specific nature of this release, we have a new changelog file with nRF54H20-specific entries, similarly to what we've done for nRF54L15.